client/src/snippets.json

```diff
@@ -0,0 +1,23 @@
+{
```

Comment:
Cool, can you add a second example so we know how this json is structured. Maybe for something like:

```
                <conditional name="output_options">
                    <param name="reads_report_type" type="select" label="What would you like to have reported?">
                        <option value="retained">The actual reads</option>
                        <option value="count">The count of reads (-c)</option>
                    </param>
                    <when value="retained">
                    </when>
                    <when value="count" />
                </conditional>
```